src/renderer/components/dex/OrderHistory.vue

```diff
@@ -22,16 +21,18 @@
           <tbody>
             <tr v-for="(order, index) in filteredOrders" :key="index">
               <td :class="['side', {green: order.side === 'Buy', red: order.side === 'Sell'}]">{{ order.side }}</td>
-              <td class="type">{{ order.type }}</td>
               <td class="market">{{ order.marketName }}</td>
               <td class="filled">{{ $formatNumber(order.quantity - order.quantityRemaining) }}</td>
               <td class="units-total">{{ $formatNumber(order.quantity) }}</td>
               <td class="price">{{ $formatNumber(order.price) }}</td>
               <td class="created">{{ $formatDateShort(order.created) }} {{ $formatTime(order.created) }}</td>
               <td class="status">
                 <div v-if="order.status === 'Open' || order.status === 'PartiallyFilled'" class="btn-cancel" @click="cancelOrder(order)">
+                  <div v-if="order.status === 'PartiallyFilled'" class="partial">
```

Comment:
This is nice, what if the status is in initial (no units filled). It seems this will show nothing. Maybe we should have it show a status of 'Inital' or something that indicates it is open with no units filled.

Reply:
Perhaps showing the status as the word 'Open' for an order that is open that has not been partially filled would suffice.